/**: # Wildcard: will load the configuration params to all nodes
  ros__parameters:
    drone:
      name: "d1"

      battery_usage_per_time_unit: # Estimated in the simulator
        searching:  0.05896 # Slow movement
        moving:     0.06201 # Movement speed of 3 m/s using internal commands

    locations:
      names: ["h1", "h2", "a1", "a2", "a3", "a4", "a5", "a6"] 
      paths:
        # Define which paths are available from each location
        h1: ["a1", "a3", "a5"]
        h2: ["a2"]
        a1: ["h1", a2]
        a2: ["h2", "a1", "a3"]
        a3: ["h1", "a2", "a4"]
        a4: ["a3", "a5"]
        a5: ["h1", "a4", "a6"]
        a6: ["a5"]
      recharge_available: ["h1", "h2"]
      resupply_available: ["h1", "h2"]

      pos_ne: # North-East
        # Helipads available
        h1: [0.0,   0.0]
        h2: [40.0,  40.0]
        
        # Areas to search
        a1: [20.0,  0.0]
        a2: [20.0,  20.0]
        a3: [0.0,   20.0]
        a4: [-20.0, 20.0]
        a5: [-20.0,  0.0]
        a6: [-20.0, -20.0]

      # Every position within location_radius assumed to be inside an area
      location_radius_m: 10.0 # Based on the area size in config.yaml 

    mission_init:
      start_location: "h1" # Currently defines the origin, which all locations based on (if using NED-frame)
      locations_available: ["h2", "a1" , "a2", "a3", "a4", "a5", "a6"] # Assuming no other drone nearby atm

      payload:
        num_markers: 2
        num_lifevests: 1

    mission_goals:
      locations_to_search: ["a1", "a2", "a3", "a4", "a5", "a6"]
      drone_landed: true
      preferred_landing_location: "h1" 
      possible_landing_locations: ["h2"] # This should be extended online


